code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -334,7 +377,10 @@
 	antag_datum = /datum/antagonist/nukeop
 	var/datum/antagonist/antag_leader_datum = /datum/antagonist/nukeop/leader
 	minimum_required_age = 14
-	restricted_roles = list(JOB_CAPTAIN, JOB_HEAD_OF_SECURITY) // Just to be sure that a nukie getting picked won't ever imply a Captain or HoS not getting drafted
+	restricted_roles = list(
+		JOB_CAPTAIN,
+		JOB_HEAD_OF_SECURITY,
+	) // Just to be sure that a nukie getting picked won't ever imply a Captain or HoS not getting drafted
```

Comment:
i'm not sure if the comment on this line is misplaced, but i think it makes sense

Reply:
i would've put it on 379 but it doesn't matter